docs/library/overlay/dialog.md

```diff
@@ -1,112 +1,13 @@
 ---
 components:
     - rx.radix.themes.DialogRoot
+    - rx.radix.themes.DialogTrigger
+    - rx.radix.themes.DialogTitle
+    - rx.radix.themes.DialogContent
+    - rx.radix.themes.DialogDescription
+    - rx.radix.themes.DialogClose
 ---
 
-```python exec
-import reflex as rx
-from reflex.components.radix.themes.components import *
-from reflex.components.radix.themes.layout import *
-from reflex.components.radix.themes.typography import *
-```
 
-# Dialog
+# Check Out Low Level Docs! High Level API Coming Soon.
```

Comment:
i'm not really a fan of requiring two clicks to get useful docs

<img width="718" alt="image" src="https://github.com/reflex-dev/reflex-web/assets/1524005/d3ab0a82-159a-4b98-91ce-96fa0be2b892">

If there is no high-level API, can we just not have that tab?

Reply:
We can for sure if there is no low level API, but it just looks a little less polished perhaps if there is only one tab for some and two for others. We can discuss and see what we want to do with this though.

Reply:
have made a change so that for all docs that do not have high level docs yet will now automatically go to the low level tab